.gitlab-ci.yml

```
@@ -12,80 +14,108 @@ workflow:
 
 default:
   interruptible: true
-  before_script:
-    # FIXME: build minimal el9 container, or use container-forge
-    - dnf install -y java-21-openjdk maven wget tree ruby python3-pip
 
 stages:
-  - build-stage
-  - test-stage
-  - deploy-stage
+  - build
+  - test
+  - deploy
 
 build:
-  stage: build-stage
+  stage: build
   script:
-    - ./build-coatjava.sh -T4 --unittests --spotbugs --quiet --no-progress
+    - ./build-coatjava.sh -T4 --quiet --no-progress
```

use dynamic number of threads from the runner constraints ([see documentation](https://pages.jlab.org/scicomp/software/code-gitlab-docs/gitlab.html#jlab-gitlab-limits))
```suggestion
    - ./build-coatjava.sh -T$JL_RUNNER_AVAIL_CPU --quiet --no-progress
```